server/api/models/post.js

```diff
@@ -26,13 +37,13 @@ class Post{
     static create(title, name, body){
         return new Promise (async (res,rej) =>{
             try{
-                let postData = await db.query(`INSERT INTO posts (title, name, body, path)
+                let postData = await db.query(`INSERT INTO posts (title, name, body)
                                                 VALUES ($1,$2,$3)
-                                                RETURNING *;`[title, name, body]);
+                                                RETURNING *;`,[title, name, body]);
                 let post = new Post(postData.rows[0])
                 res(post);                   
             }catch (err){
-                rej('counldnt create')
+                rej(`counldnt create because of ${err}`)
             }
         })
     }
```

Comment:
great model, great queries, works perfectly.

Reply:
thanks man
we're proud of it